### PR TITLE
Add specter file and deps

### PR DESCRIPTION
**Pull request by pfarwick-latacora**
Making draft PR for adding specter training to koans. 

Will hopefully be updating every day so not going to merge just yet

### CODE REVIEW

File: src/koans/28_specter.clj [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+                   :rav4  [{:doors 4
+                            :color "blue"}
+                           {:doors 4
+                            :color "blue"}]}
+          :latacara {:prime [{:doors 11
+                              :color "clear"}]
+                     :prime-zero [{:doors 0
+                                   :color "red"}
+                                  {:doors 0
+                                   :color "green"}]}}})
```
Comment (lvh) [inline src/koans/28_specter.clj]: No Guards Red 1986 911 Turbo?

Unacceptable.

**Review by lvh — CHANGES_REQUESTED** [review]
Never used Koan engine before, looks sweet! Also, no 911, unacceptable, what the fuck

**Comment by latacora-lizzie** [comment]
:+1: exciting!

**Comment by pfarwick-latacora** [comment]
@lvh as if you would ever sell the 911 at our make believe car dealership. 

How dare you actually!

**Comment by pfarwick-latacora** [comment]
I am sure we can add more to this in the future but I think it gets people familiar enough with specter and eidolon in order to understand / work on edit.clj files in a meaningful enough way.

**Review by lvh — APPROVED** [review]

**Pull request merged**